src/services/systemsManagerInstance/schema.graphql

```diff
@@ -16,17 +16,17 @@ type awsSystemsManagerInstance implements awsBaseService @key(fields: "arn") {
   lastSuccessfulAssociationExecutionDate: DateTime @search(by: [day])
   ipAddress: String @search(by: [hash, regexp])
   computerName: String @search(by: [hash, regexp])
-  associationOverview: systemsManagerInstanceAssociationOverview
+  associationOverview: systemsManagerInstanceAssociation
   complianceItems: [systemsManagerInstanceComplianceItem]
   sourceId: String @search(by: [hash, regexp])
   sourceType: String @search(by: [hash, regexp])
   iamRole: [awsIamRole] @hasInverse(field: systemsManagerInstances)
   ec2Instance: [awsEc2] @hasInverse(field: systemsManagerInstance)
 }
 
-type systemsManagerInstanceAssociationOverview {
+type systemsManagerInstanceAssociation {
   detailedStatus: String @search(by: [hash, regexp])
-  instanceAssociationStatusAggregatedCount: [ssmAssociationOverviewAggregatedCount]
+  instanceAssociationCount: [ssmAssociationOverviewAggregatedCount]
```

Comment:
does `aggCount` work here?

Reply:
ok, I'm not sure. But I'm going to try it

Reply:
this can work like this `ssmAssociationAggCount`. What do you think?

Reply:
probably fine yea